controllers/kyma_controller.go

```diff
@@ -104,6 +102,13 @@ func (r *KymaReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.
 		return ctrl.Result{}, client.IgnoreNotFound(err) //nolint:wrapcheck
 	}
 
+	if status.SetConditions(kyma, r.WatcherEnabled(kyma)) {
+		if err := r.UpdateStatus(ctx, kyma, kyma.Status.State, "Initialising Conditions"); err != nil {
```

Comment:
Here you don't need to update the status immediately, just need to have an initialized conditions, in the `stateHandling`, update status will happen eventually anyway.